docs/administer/rbac.md

```diff
@@ -0,0 +1,435 @@
+
+
+    ### Key considerations for RBAC
+
+    Before you start defining the different roles, there are some important things to consider when it comes to Role-Based Access Control (RBAC).
+    {.power-number}
+
+    === "Namespaces"
+
+        Read `namespaces` permissions are required for **all** the **roles**. 
+
+    === " DB clusters"
+    
+        To create, update and delete database clusters, backups, schedules, monitoring instances, and restores it is recommended to **explicitly** grant `read` permissions as well for the `database clusters`. Without these permissions, you would not be able to view these resources, which would not be practical.
+
+    === "Resources"
+    
+        To create, update and delete the resources, it is recommended to **explicitly** grant `read` permissions for these resources as well. Without these permissions, you would not be able to view these resources, which would not be practical.
+
+        Example: To manage `backup schedules` (create, update, delete a schedule), it is recommended to **explicitly** grant `read` permissions for the `backup schedules` as well as `backup storages`.
+
+    === "Backups"
+    
+        For on-demand backups and schedules, you should grant `read` permissions for `backup storages` as well.
+
+
+    === "Restores"
+
+        For restores, to new and existing databases, you should grant the following permisssions as well:
+
+        - Read `backups` (of the old DB)
+        - Read `MonitoringConfig` (if the old DB has monitoring enabled)
+        - Read `database cluster credentials` (of the old DB)
+        - Create `backups` (if backup schedules are enabled)
+   
+    === "Upgrades"
+
+        For upgrades, the following permissions must be granted:
+
+        - Read Namespaces
+        - Read all `database engines` in that namespace
+        - Read all `database clusters` in that namespace
+        - Update `database clusters` in that namespace
+
+
+
+
+## Roles in RBAC
+
+In Role-Based Access Control (RBAC), a Role is a set of permissions that define what actions (like read, write, update, delete) can be performed on specific resources within Percona Everest. In RBAC, roles are assigned to users, allowing them to interact with the resources according to the permissions defined by their roles.
+
+### Default role
+
+In Percona Everest, the only predefined role is **adminrole:role**, which is assigned to the **admin** user. Hence, this **admin** user has unrestricted access to Percona Everest. However, the RBAC (Role-Based Access Control) configuration can define and allocate specific roles based on individual requirements and access privileges.
+
+This default built-in role definition can be seen in [policy.csv](https://github.com/percona/everest/blob/main/deploy/quickstart-k8s.yaml#L94-L102).
+
```

Comment:
No longer exists here
```suggestion
```

Reply:
Additionally, it might be worth mentioning that the `role:admin` always exists internally. You can not see it in the the ConfigMap, hence you cannot modify it either. You may only assign roles/users to it.